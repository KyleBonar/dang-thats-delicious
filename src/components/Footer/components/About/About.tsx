import * as React from "react";

import styled from "../../../../theme/styled-components";

const StyledDiv = styled.div`
  max-width: ${props => props.theme.footerMaxWidth};
`;

const StyledH5 = styled.h5`
  margin: 0.5em 0;
  font-weight: 400;
  padding: 0;
  text-transform: uppercase;
  color: ${props => props.theme.white};
  font-size: ${props => props.theme.scaleH5};
`;

const StyledP = styled.p`
  margin: 0.5em 0;
  padding: 0;
  color: ${props => props.theme.white};
  font-size: ${props => props.theme.scaleP};
`;

interface TypesProps {
  className?: string;
}

const Types: React.SFC<TypesProps> = props => {
  return (
    <StyledDiv className={props.className}>
      <StyledH5>About</StyledH5>

      <StyledP>Text here</StyledP>
    </StyledDiv>
  );
};

export default Types;
