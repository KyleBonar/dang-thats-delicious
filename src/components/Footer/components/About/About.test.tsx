import * as React from "react";
import * as enzyme from "enzyme";

import About from "./About";

describe("<About />", () => {
  it("renders", () => {
    const wrapper = enzyme.shallow(<About />);
    expect(wrapper).toBeTruthy();
  });

  it("matches snapshot", () => {
    const wrapper = enzyme.shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders single h5 tag", () => {
    const wrapper = enzyme.shallow(<About />);
    expect(wrapper.find("h5").length).toEqual(1);
  });

  it("renders single p tag", () => {
    const wrapper = enzyme.shallow(<About />);
    expect(wrapper.find("p").length).toEqual(1);
  });
});
