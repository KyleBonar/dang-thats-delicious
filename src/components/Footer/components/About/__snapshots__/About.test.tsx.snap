// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> matches snapshot 1`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 2`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 3`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 4`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 5`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 6`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 7`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 8`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 9`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 10`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 11`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 12`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 13`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 14`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 15`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 16`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 17`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 18`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 19`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 20`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 21`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 22`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 23`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 24`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 25`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 26`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 27`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 28`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 29`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 30`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 31`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;

exports[`<About /> matches snapshot 32`] = `
<div>
  <h5>
    About
  </h5>
  <p>
    Ever wonder what a sauce tastes like 
    <em>
      before
    </em>
     buying it? Sauce For Your Thoughts can help! Add, or review, sauces and help out your fellow saucies! Never be blind-sides by a sauce again!
  </p>
</div>
`;
